pkg/ignition/builder.go

```diff
@@ -99,10 +102,15 @@ func (b *ignitionBuilder) Generate() ([]byte, error) {
 		"/etc/NetworkManager/conf.d/clientid.conf",
 		0644, false,
 		[]byte("[connection]\nipv6.dhcp-duid=ll\nipv6.dhcp-iaid=mac")))
+
+	update_hostname := fmt.Sprintf(`
+	[[ \"$DHCP6_FQDN_FQDN\" =~ \".\" ]] && hostnamectl set-hostname --static --transient $DHCP6_FQDN_FQDN 
```

Comment:
nit: I assume the escapes on the double quotes are still OK, but no longer necessary.

Reply:
yeah, I didn't want to tough the existing DHCP6 FQDN based update to the hostname.

Reply:
Right, but the `\` was for golang because it was inside a double-quoted string, but now it's inside a backtick string there's no need for escaping afair.